src/common/utils/time/time-utils.ts

```diff
@@ -0,0 +1,5 @@
+import moment from 'moment';
+
+export function formatDateTime(date: Date, formatString: string): string {
```

Comment:
Consider adding a comment with a link to the format string options from the moment documentation